react/week4/github_user_search/src/App.js

```diff
@@ -0,0 +1,14 @@
+import "./App.css";
+import User from "./consumeUser.js";
+import UserRepos from "./githubApi.js";
```

Comment:
(see my two comments below). here you could name your components more simple names:

    import User from "./User.js";
    import UserRepos from "./UserRepos.js";

Reply:
sure kasper, Thanks :)